src/directions/distance_matrix_service.ts

```diff
@@ -24,7 +28,7 @@ const KILOMETERS_TO_METERS_CONSTANT = 1000;
 export class MigrationDistanceMatrixService {
   // This will be populated by the top level module
   // that creates our location client
-  _client: LocationClient;
+  _client: GeoRoutesClient;
 
   // This will be populated by the top level module
   // that is passed our route calculator name
```

Comment:
We can remove this line about the route calculator now that it will be removed as part of this change :)